woocommerce/woocommerce-ios
# Reviews: fixed dark mode appearance

Fixes #1385 

The scope of the PR is limited to just making the Reviews screen usable in Dark mode when building the codebase with the release version of Xcode 11.
The dark mode appearance on "Loading review list" screen and "Review list" screen was already fixed.

## Testing
- Activate dark mode
- Navigate to Product reviews
- Navigate from to a Review detail
- Check that everything is visually ok (readable, and without dark background)

## Screenshots

| Before             |  After |
:-------------------------:|:-------------------------:
![67190779-2a2aee00-f423-11e9-8aaf-ea5e8fa6519b](https://user-images.githubusercontent.com/495617/68495049-bb85c700-024f-11ea-9cf8-427a6dc607e3.png) | ![Simulator Screen Shot - iPhone 11 Pro - 2019-11-08 at 17 46 01](https://user-images.githubusercontent.com/495617/68495059-bfb1e480-024f-11ea-8118-4f1986142f54.png)


Update release notes:
